xwiki-rendering-api/src/main/java/org/xwiki/rendering/listener/chaining/AbstractChainingListener.java

```diff
@@ -67,23 +80,39 @@
      */
     public AbstractChainingListener()
     {
+        this.listItemRetroCompatibility = needsRetroCompatibility(method -> method.getName().endsWith("ListItem"), 0);
+        this.imageRetroCompatibility = needsRetroCompatibility(method -> method.getName().equals("onImage"), 3);
+    }
+
+    /**
+     * @param methodFilter A predicate to filter the methods.
```

Comment:
param text usually start with a lower case.

I've been pointed to https://blog.joda.org/2012/11/javadoc-coding-standards.html for our javadoc style (a long time ago, I'm not sure it's actually in our documentation).

>  **Treat @param and @return as a phrase**
> 
> The @param and @return should be treated as phrases rather than complete sentences. They should start with a lower case letter, typically using the word "the". They should not end with a dot. This aids readability in source code and when generated.

Reply:
[Our documentation](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HJavadocBestPractices) mentions the [Oracle Javadoc coding conventions](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide) which mentions phrases but doesn't require them. I think I got somehow too much into writing comments as sentences as we prefer them for the initial sentence and also for inline comments. I've fixed this in another commit, I hope I found all of them.